services/QuillLMS/spec/models/teacher_activity_feed_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe TeacherActivityFeed, type: :model do
```

Comment:
There were additional tests in the test file for the helper that I wrote that we probably want to move over here.

Reply:
> There were additional tests in the test file for the helper that I wrote that we probably want to move over here.

Thanks for flagging, will do!